vault/logical_raw.go

```diff
@@ -353,11 +351,6 @@ func rawPaths(prefix string, r *RawBackend) []*framework.Path {
 				},
 				logical.CreateOperation: &framework.PathOperation{
 					Callback: r.handleRawWrite,
-					DisplayAttrs: &framework.DisplayAttributes{
-						OperationPrefix: "raw",
-						OperationVerb:   "write",
-						OperationSuffix: "|path",
-					},
```

Comment:
Yes, deliberate - this is a CreateOperation, so it's wholly ignored for OpenAPI purposes (beyond setting x-vault-createSupported) in favour of the information from the UpdateOperation.

If it wasn't ignored like that, it would have ended up generating duplicate operation IDs, the way it was previously written.